BrowserKit/Sources/SiteImageView/ImageProcessing/BundleImageFetcher/BundleImageFetcher.swift

```diff
@@ -34,8 +34,9 @@ class DefaultBundleImageFetcher: BundleImageFetcher {
         bundledImages = retrieveBundledImages()
     }
 
-    func getImageFromBundle(domain: ImageDomain) throws -> UIImage {
-        guard let bundleDomain = getBundleDomain(domain: domain) else {
+    func getImageFromBundle(domain: ImageDomain?) throws -> UIImage {
+        guard let domain = domain,
+              let bundleDomain = getBundleDomain(domain: domain) else {
```

Comment:
nit: `else` should be on a separate line if multi-line guards. And because the else clause is a single line, we can have that be a single line too. :)